environments/base/manifests/thanos-store-shard0-statefulSet.yaml

```diff
@@ -51,6 +51,28 @@ spec:
             "max_item_size": "1MiB"
             "timeout": "2s"
           "type": "MEMCACHED"
+        - |-
+          --store.caching-bucket.config="blocks_iter_ttl": "5m"
+          "chunk_object_attrs_ttl": "24h"
+          "chunk_subrange_size": 16000
+          "chunk_subrange_ttl": "24h"
+          "config":
+            "addresses":
+            - "dnssrv+_client._tcp.observatorium-xyz-thanos-store-memcached.observatorium.svc"
+            "dns_provider_update_interval": "10s"
+            "max_async_buffer_size": 100000
+            "max_async_concurrency": 100
+            "max_get_multi_batch_size": 1000
+            "max_get_multi_concurrency": 900
+            "max_idle_connections": 1000
+            "max_item_size": "1MiB"
```

Comment:
Can we ask Marco about their settings here? I can't believe that 1MiB is a very good size for this.

Reply:
Yes, I can do that. FYI, these are the defaults from upstream Thanos.

Reply:
Yeah we should improve those defaults as well.

Reply:
@brancz After some offline conversation and checking the PR again, values seem alright. Caching bucket caches the chunk pieces using `Bucket.GetRange` so the cached pieces are a lot smaller than `max_item_size` limit. And the limit is specified by `chunk_subrange_size: 16000`. So it's good to go. We can do fine-tuning at downstream when needed.

Reply:
Got it, thanks for clarifying. All good from my side!